apis/infra/v1alpha1/cluster_types.go

```diff
@@ -28,11 +28,6 @@ type Cluster struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	// ScaleProfileName contains the name of a ClusterScaleProfile in the same namespace,
-	// which provides cluster scaling characteristics
-	// +optional
-	ScaleProfileName *string `json:"scaleProfileName,omitempty"`
```

Comment:
Probably should clean up the manifests in the config dir too, to match these changes.

Reply:
done, reran `make generate` and updated the files in `config/samples`